js/packages/web/src/views/pack/components/PackSidebar/index.tsx

```diff
@@ -15,12 +15,13 @@ interface IPropsPackSidebar {
 }
 
 const PackSidebar = ({ onOpenPack }: IPropsPackSidebar) => {
-  const { pack, voucherMetadataKey } = usePack();
+  const { pack, voucherMetadataKey, cardsRedeemed } = usePack();
 
   const metadataPubkey = voucherMetadataKey || '';
   const art = useArt(metadataPubkey);
   const { publicKey } = useWallet();
   const userWallet = pubkeyToString(publicKey);
+  const canOpen = pack?.info.allowedAmountToRedeem !== cardsRedeemed;
```

Comment:
There is a const to determine if pack is already fully opened

```suggestion
  const isExhausted = provingProcess?.info.isExhausted
```